src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs

```diff
@@ -114,7 +115,17 @@ internal sealed class ODataJsonLightOutputContext : ODataOutputContext
 
                 // COMPAT 2: JSON indentation - WCFDS indents only partially, it inserts newlines but doesn't actually insert spaces for indentation
                 // in here we allow the user to specify if true indentation should be used or if the limited functionality is enough.

```

Comment:
Going to remove this comment. It doesn't seem to apply to the current codebase. It used to be a applicable when there was an EnableIndentation() method to the writer settings, but that was removed 3-6 years ago.

https://github.com/OData/odata.net/blame/e3de4760cc3eb33a08a938900e20e2f3fabff1c6/src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs#L89